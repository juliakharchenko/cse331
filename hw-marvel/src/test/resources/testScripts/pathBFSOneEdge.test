# Tests finding a path with only one edge between two nodes.

LoadGraph nba nbaPlayers.csv

FindPath nba MICHAEL-JORDAN DWYANE-WADE