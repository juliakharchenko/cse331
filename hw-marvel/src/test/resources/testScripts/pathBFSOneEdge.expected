# Tests finding a path with only one edge between two nodes.

loaded graph nba

path from MICHAEL-JORDAN to DWYANE-WADE:
MICHAEL-JORDAN to DWYANE-WADE via BULLS